test/test_base_registrar.js

```diff
@@ -135,7 +135,10 @@ contract('BaseRegistrar', function (accounts) {
 		var expires = await registrar.nameExpires(sha3("newname"));
 		var grace = await registrar.GRACE_PERIOD();
 		await advanceTime(expires.toNumber() - ts + grace.toNumber() + 3600);
```

Comment:
@Arachnid please use the advance time from our test helper library.

Reply:
@decanus We're not currently using testUtils here; I think refactoring this entire file would be out-of-scope.